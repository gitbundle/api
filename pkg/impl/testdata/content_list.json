[
    {
        "name": "advanced.en-us.md",
        "path": "docs/content/doc/advanced.en-us.md",
        "sha": "de2bfeed6f2d3ea1d2f91e457e9450efc582bb31",
        "type": "file",
        "size": 192,
        "encoding": null,
        "content": null,
        "target": null,
        "url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.en-us.md?ref=master",
        "html_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.en-us.md",
        "git_url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/de2bfeed6f2d3ea1d2f91e457e9450efc582bb31",
        "download_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.en-us.md",
        "submodule_git_url": null,
        "_links": {
            "self": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.en-us.md?ref=master",
            "git": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/de2bfeed6f2d3ea1d2f91e457e9450efc582bb31",
            "html": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.en-us.md"
        }
    },
    {
        "name": "advanced.fr-fr.md",
        "path": "docs/content/doc/advanced.fr-fr.md",
        "sha": "04734a8f81f5641de2b6d738bc9b777c2cc541c6",
        "type": "file",
        "size": 190,
        "encoding": null,
        "content": null,
        "target": null,
        "url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.fr-fr.md?ref=master",
        "html_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.fr-fr.md",
        "git_url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/04734a8f81f5641de2b6d738bc9b777c2cc541c6",
        "download_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.fr-fr.md",
        "submodule_git_url": null,
        "_links": {
            "self": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.fr-fr.md?ref=master",
            "git": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/04734a8f81f5641de2b6d738bc9b777c2cc541c6",
            "html": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.fr-fr.md"
        }
    },
    {
        "name": "advanced.zh-cn.md",
        "path": "docs/content/doc/advanced.zh-cn.md",
        "sha": "1f7ebf81e723a0ba82a89eec18a2706ac1b0dace",
        "type": "file",
        "size": 188,
        "encoding": null,
        "content": null,
        "target": null,
        "url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.zh-cn.md?ref=master",
        "html_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.zh-cn.md",
        "git_url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/1f7ebf81e723a0ba82a89eec18a2706ac1b0dace",
        "download_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/raw/branch/master/docs/content/doc/advanced.zh-cn.md",
        "submodule_git_url": null,
        "_links": {
            "self": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced.zh-cn.md?ref=master",
            "git": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/1f7ebf81e723a0ba82a89eec18a2706ac1b0dace",
            "html": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced.zh-cn.md"
        }
    },
    {
        "name": "advanced",
        "path": "docs/content/doc/advanced",
        "sha": "8c43a4fb023735dfd026014bae92ad91414b88bd",
        "type": "dir",
        "size": 0,
        "encoding": null,
        "content": null,
        "target": null,
        "url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced?ref=master",
        "html_url": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced",
        "git_url": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8c43a4fb023735dfd026014bae92ad91414b88bd",
        "download_url": null,
        "submodule_git_url": null,
        "_links": {
            "self": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/contents/docs/content/doc/advanced?ref=master",
            "git": "https://try.gitea.io/api/v1/repos/aaaaaz/go-gitea.gitea/git/blobs/8c43a4fb023735dfd026014bae92ad91414b88bd",
            "html": "https://try.gitea.io/aaaaaz/go-gitea.gitea/src/branch/master/docs/content/doc/advanced"
        }
    }
]