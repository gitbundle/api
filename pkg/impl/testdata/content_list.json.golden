[
  {
    "path": "docs/content/doc/advanced.en-us.md",
    "kind": "file",
    "blobid": "de2bfeed6f2d3ea1d2f91e457e9450efc582bb31"
  },
  {
    "path": "docs/content/doc/advanced.fr-fr.md",
    "kind": "file",
    "blobid": "04734a8f81f5641de2b6d738bc9b777c2cc541c6"
  },
  {
    "path": "docs/content/doc/advanced.zh-cn.md",
    "kind": "file",
    "blobid": "1f7ebf81e723a0ba82a89eec18a2706ac1b0dace"
  },
  {
    "path": "docs/content/doc/advanced",
    "kind": "directory",
    "blobid": "8c43a4fb023735dfd026014bae92ad91414b88bd"
  }
]
